src/main/resources/api.yaml

```diff
@@ -1470,7 +1478,12 @@ definitions:
         type: array
         description: The list of the roles of the user
         items:
-          type: string
+          $ref: '#/definitions/RoleDetail'
+      userGroup:
+        type: array
```

Comment:
This is incorrect.  `userGroup` is not an array, its an object.  See https://github.com/hmcts/idam-app/blob/master/api/src/main/java/uk/gov/hmcts/idam/endpoint/auth/IntegrationTestSupportEndpoint.java#L92